modules/assignment/src/main/java/com/intuit/wasabi/assignment/Assignments.java

```diff
@@ -180,6 +180,19 @@ boolean doSegmentTest(Application.Name applicationName, Experiment.Label experim
                           HttpHeaders headers);
 
     /**
+     * This method is used to clear assignments metadata cache.
+     *
+     * @return True if cache is cleared successfully
+     *
+     */
+    void clearMetadataCache();
+
+    /**
+     * Clear metadata cache if it is enabled.
```

Comment:
Should this comment be get metadata cache details and not clear metadata cache ?